### ISSUE TITLE
Android not rendering Pie

### ISSUE THREAD

**Issue by shawna-donnelly** [nihgwu/react-native-pie#33]
Is there something particular I need to do to get Pie to render in Android? It works fine in iOS, but Android will not render the pie. I know it will render other items (I'm using it for markers in react-native-maps), because I have gotten those to show up, but Pie always comes back with nothing. There is no block or anything like that, it's just not there. Any help is appreciated.

**Comment by JoaoPauloCMarra** [comment]
same happens for me

**Comment by zgordon01** [comment]
Can you post a code snippet? Also what version of react native are you running?

**Comment by CharlyPoppins** [comment]
@shawna-donnelly problem solved ?

**Comment by scannbook** [comment]
@CharlyPoppins same issue for me. I use react-native 0.63

**Comment by shawna-donnelly** [comment]
@CharlyPoppins no, this has not been resolved. I ended up switching to a new library, but that one is a lot less performant than this one. I'd much prefer using this library. It does not render the Pie in Android at all. I thought it might have been something to do with the colors passed, but that doesn't seem to make a difference. I passed "red" and a color code and it doesn't make a difference. It works perfectly on iOS, but does not render at all on Android.

**Comment by CharlyPoppins** [comment]
Hello, I switched to victory-native https://formidable.com/open-source/victory/docs/native/
